packages/web3-providers/src/Twitter/apis/getUserViaWebAPI.ts

```diff
@@ -11,6 +11,7 @@ const features = {
 }
 async function createRequest(screenName: string) {
     const { bearerToken, csrfToken, queryId } = await getTokens('UserByScreenName')
```

Comment:
Reflect the truth of `undefined` in the return type.

Reply:
Already Reflected.
<img width="342" alt="image" src="https://user-images.githubusercontent.com/63733714/224266832-559ccae4-4351-40c8-92a1-29f2e22898d1.png">